src/main/kotlin/com/cognifide/gradle/aem/environment/hosts/HostsOptions.kt

```diff
@@ -0,0 +1,26 @@
+package com.cognifide.gradle.aem.environment.hosts
+
+import org.apache.commons.lang3.SystemUtils
+
+class HostsOptions {
+    var list = listOf(
```

Comment:
I am not sure if sth should be here

Reply:
this mean GAP strictly related to gradle-aem-multi which cannot be done

Reply:
I moved default dispatcher configuration to GAP, then it gets expanded. This way it is strictly GAP feature which configuration can be overridden in the specific project.